src/components/ProgressTable/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import { ProgressbarCard } from './ProgressbarCard';
+
+import { Title } from '../Title';
+
+import ProgressBarLevel1 from '@/../public/images/progressbar-bg-level-1.png';
+import ProgressBarLevel2 from '@/../public/images/progressbar-bg-level-2.png';
+import ProgressBarLevel3 from '@/../public/images/progressbar-bg-level-3.png';
+
+export const Progressbar = () => {
+  return (
+    <>
+      <Title>Progress</Title>
+      <div className="ml-9 mt-9 flex gap-x-9">
+        <ProgressbarCard
```

Comment:
Create array with such objects and then just map over them

Example:
```js
const array = [{ title: "A", image: Img, amount: 228, maxAmount: 1337 }];

return array.map(({ title, image, amount, maxAmount }) => <ProgressbarCard key={title} title={title} .... /> )

Reply:
Does the best practice do mapping in jsx markup ?